src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+is_a: MONDO:0005583 {source="OMIA:000038"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0019439 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog AA amyloidosis
+
+[Term]
+id: MONDO:1010093
+name: AL amyloidosis, non-human animal
+def: "AL amyloidosis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000039 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000039"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0019438 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog AL amyloidosis
+
+[Term]
+id: MONDO:1010094
+name: analphalipoproteinaemia, non-human animal
+def: "Analphalipoproteinaemia that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000042 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000042"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000042"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0008783 {source="OMIA:000042"} ! cross-species analog Tangier disease
+
+[Term]
+id: MONDO:1010095
+name: citrullinaemia, non-human animal
+def: "Citrullinaemia that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000194 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000194"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000194"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0008988 {source="OMIA:000194"} ! cross-species analog citrullinemia type I
+
+[Term]
+id: MONDO:1010096
+name: glycogen storage disease Ia, non-human animal
+def: "Glycogen storage disease Ia that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000418 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000418"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000418"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009287 {source="OMIA:000418"} ! cross-species analog glycogen storage disease due to glucose-6-phosphatase deficiency type IA
+
+[Term]
+id: MONDO:1010097
+name: glycogen storage disease VII, non-human animal
+def: "Glycogen storage disease VII that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000421 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000421"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000421"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009295 {source="OMIA:000421"} ! cross-species analog glycogen storage disease VII
+
+[Term]
+id: MONDO:1010098
+name: maple syrup urine disease BCKDHA-realted, non-human animal
```

Comment:
- typo in "related"
- shall we ignore this general species unspecific gene-related term, and instead add the species- specific term?

Reply:
yes